PARAM_DIFFERENCES.md

```diff
@@ -0,0 +1,57 @@
+| columnDimensions          | Dimensions of the column space                                                                                                                                                                                                                                                                                                                                                                                       | \-          | columnDimensions          |             | \-2048  |
+| cellsPerColumn            | Number of cells per column                                                                                                                                                                                                                                                                                                                                                                                           | 32          | cellsPerColumn            |             | 32      |
+| activationThreshold       | If the number of active connected synapses on a segment is at least this threshold, the segment is said to be active\.                                                                                                                                                                                                                                                                                               | 13          | activationThreshold       |             | 13      |
+| initialPermanence         | Initial permanence of a new synapse\.                                                                                                                                                                                                                                                                                                                                                                                | 0\.21       | initialPermanence         |             | 0\.21   |
+| connectedPermanence       | If the permanence value for a synapse is greater than this value, it is said to be connected\.                                                                                                                                                                                                                                                                                                                       | 0\.5        | connectedPermanence       |             | 0\.5    |
```

Comment:
nit: probobly we won't cover that here, but there's a diff between what's in the defaults param, and what's actually used as a default everywhere (tests, NAB,..). For connectedPermanence it's 0.2 for Numenta.

Reply:
I have another column "Value in NAB" for both, HTMCore as well as Numenta's NuPiC, but I removed it. I can add it back into the table, but I would propose to wait until we have the param set for HTMCore optimised.

Reply:
> I have another column "Value in NAB" for both 

yes, "NAB column" is a good idea! 
And yes, let's wait where we get there.